lms/static/scripts/frontend_apps/components/ContentSelector.tsx

```diff
@@ -164,6 +169,11 @@ export default function ContentSelector({
     onSelectContent({ type: 'file', file });
   };
 
+  const selectCanvasPage = (page: Page) => {
+    cancelDialog();
+    onSelectContent({ type: 'url', url: page.id });
```

Comment:
Selecting the page itself doesn't work yet. That's out of this PR scope.

We probably need to change this later, once aligned with the BE.

Reply:
This should work now for the most basic cases in `main`.